generate_so.py

```diff
@@ -514,4 +515,133 @@ def _make(dirpath: str) -> None:
+            SoConfig(36,  0.65, 12.5, 8.0, 10.0, 2.0, 'GC'),
+            SoConfig(40,  0.65, 14.0, 8.0, 10.0, 2.0, 'GD'),
+            #   0.5mm pitch
+            SoConfig(36,  0.50,  9.7, 8.0, 10.0, 2.0, 'HA'),
+            SoConfig(40,  0.50, 11.0, 8.0, 10.0, 2.0, 'HB'),
+            SoConfig(48,  0.50, 12.5, 8.0, 10.0, 2.0, 'HC'),
+            SoConfig(56,  0.50, 14.0, 8.0, 10.0, 2.0, 'HD'),
+            #   0.4mm pitch
+            SoConfig(48,  0.40,  9.7, 8.0, 10.0, 2.0, 'JA'),
+            SoConfig(52,  0.40, 11.0, 8.0, 10.0, 2.0, 'JB'),
+            SoConfig(56,  0.40, 12.5, 8.0, 10.0, 2.0, 'JC'),
+            SoConfig(60,  0.40, 12.5, 8.0, 10.0, 2.0, 'JC-1'),
+            SoConfig(64,  0.40, 14.0, 8.0, 10.0, 2.0, 'JD'),
+            SoConfig(68,  0.40, 14.0, 8.0, 10.0, 2.0, 'JD-1'),
+        ],
+        lead_width_lookup={
+            0.65: 0.30,
+            0.50: 0.27,
+            0.40: 0.23,
+        },
+        lead_contact_length=0.6,
+        pkgcat='3627bf02-2e6e-4d68-9ada-743fa69a4f8c',
+        keywords='so,sop,ssop,small outline package,smd,jedec,mo-152',
+        version='0.1',
+        create_date='2019-07-21T12:55:20Z',
+    )
+    generate_pkg(
+        dirpath='out/ssop/pkg',
+        author='Danilo B.',
+        # Name according to IPC7351C
+        name='SSOP{pin_count}P{pitch}_{body_length}X{lead_span}X{height}L{lead_length}X{lead_width}',
+        description='{pin_count}-pin Plastic Shrink Small Outline Package (SSOP), '
+                    'standardized by JEDEC (MO-150), variation {variation}.\\n\\n'
+                    'Pitch: {pitch:.2f} mm\\nBody length: {body_length:.2f} mm\\n'
+                    'Body width: {body_width:.2f} mm\\nLead span: {lead_span:.2f} mm\\n'
+                    'Height: {height:.2f} mm\\n'
+                    'Lead length: {lead_length:.2f} mm\\nLead width: {lead_width:.2f} mm',
+        configs=[
+            # pin count, pitch, body length, body width, total width, height
+
+            # Symbols based on JEDEC MO-150:
+            #        N   e      D    E1   E    A
+
+            SoConfig( 8, 0.65,  3.0, 5.3, 7.8, 2.0, 'AA'),
+            SoConfig(14, 0.65,  6.2, 5.3, 7.8, 2.0, 'AB'),
+            SoConfig(16, 0.65,  6.2, 5.3, 7.8, 2.0, 'AC'),
+            SoConfig(18, 0.65,  7.2, 5.3, 7.8, 2.0, 'AD'),
+            SoConfig(20, 0.65,  7.2, 5.3, 7.8, 2.0, 'AE'),
+            SoConfig(22, 0.65,  8.2, 5.3, 7.8, 2.0, 'AF'),
+            SoConfig(24, 0.65,  8.2, 5.3, 7.8, 2.0, 'AG'),
+            SoConfig(28, 0.65, 10.2, 5.3, 7.8, 2.0, 'AH'),
+            SoConfig(30, 0.65, 10.2, 5.3, 7.8, 2.0, 'AJ'),
+            SoConfig(38, 0.65, 12.6, 5.3, 7.8, 2.0, 'AK'),
+        ],
+        lead_width_lookup={
+            0.65: 0.38,
+        },
+        lead_contact_length=0.75,
+        pkgcat='3627bf02-2e6e-4d68-9ada-743fa69a4f8c',
+        keywords='so,sop,ssop,small outline package,smd,jedec,mo-150',
+        version='0.1',
+        create_date='2019-07-21T12:55:20Z',
```

Comment:
Just a small detail, but should we use a more current date?

Reply:
Doesn't really matter, it's the date when I started working on those parts.

As far as I know @ubruhin even thought about removing the date completely. It doesn't have any function.

Reply:
Yeah, seeing a last updated date (gut log) may be interesting to see if the library is still maintained, but created date is a bit moot.